src/main/java/InSuffArgsException.java

```diff
@@ -0,0 +1,3 @@
+public class InSuffArgsException extends Exception {
+
+}
```

Comment:
I'm confused about the need for this exception class. Why not just throw an Exception if there is no overriding error message